devicon.json

```diff
@@ -4248,6 +4248,31 @@
         "color": "#1FA6CA",
         "aliases": []
     },
+    {
+        "name": "materializecss",
```

Comment:
It's not inserted at the right place alphabetically. It should be above `materialui` since *material`i`ze* comes before *material`u`i*